core/domain/summary_services.py

```diff
@@ -493,6 +493,12 @@ def get_library_groups(language_codes):
             '" OR "'.join(language_codes))
 
     def _generate_query(categories):
+        """Generates query based on the categories and language codes.
+        Args:
```

Comment:
Ditto. Leave one blank line.

Reply:
Done!

Reply:
Still this comment is not completely resolve. PTAL!